tests/Unit/Numerical/RootFinding/Test_QuadraticEquation.cpp

```diff
@@ -0,0 +1,15 @@
+
+#include <catch.hpp>
+
+#include "Numerical/RootFinding/QuadraticEquation.hpp"
+
+TEST_CASE("Unit.Numerical.RootFinding.PositiveRoot",
+          "[Numerical][RootFinding][Unit]") {
+  CHECK(Approx(6.31662479035539985) == positive_root(1.0, -6.0, -2.0));
+}
+
```

Comment:
add a test that has two equal positive roots and triggers the assert
add a test that has two positive roots and triggers the assert
add a test that has no positive roots and triggers the assert